lib/index.js

```diff
@@ -108,26 +124,50 @@ FSTree.prototype = {
     }
   },
 
-  get _changes() {
-    return this._parent ? this._parent._changes : this.__changes;
+  get exclude() {
+    return this._parent ? this._parent._exclude : this._exclude;
```

Comment:
Updated all the parent to use the getter, thanks for the reminder :)

Will be adding grandparents tests as well.